plugin/adsk/plugin/plugin.cpp

```diff
@@ -315,6 +320,13 @@ MStatus initializePlugin(MObject obj)
         }
     }
 
+#if defined(WANT_UFE_BUILD)
+    MayaUsd::LayerManager::addSupportForNodeType(MAYAUSD_NS::ProxyShape::typeId);
+#if defined(WANT_QT_BUILD)
+    MayaUsd::LayerManager::SetBatchSaveDelegate(UsdLayerEditor::batchSaveLayersUIDelegate);
+#endif
```

Comment:
Hooks up the callback to provide a UI to choose names/locations for anonymous layers during save.